pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -49,10 +57,39 @@
       #mobile-header.mobile-headers.mobile-feature
         = view :mobile_header_responsive
 
-  - if show_single_hero
+  - if show_single_hero == "dance2019"
+    #actions.mobile-center.col-50
+      = main_actions
+
+    -# Stars image and text below it
+    %a{href: "/dance"}
```

Comment:
confirming that we're revamping the current /dance page rather than having this go to a new /dance-2019?

Reply:
Confirmed!